code/modules/mob/living/living.dm

```diff
@@ -386,6 +389,10 @@
 			return
 
 	if(ismovableatom(A))
+		if(isxeno(src))
+			var/mob/living/carbon/xenomorph/meanie = src
+			if(meanie.xeno_caste.caste_flags & CASTE_CANT_PUSH && (istype(A,/mob/living/carbon/human) || istype(A, /mob/living/carbon/xenomorph)) && (a_intent == INTENT_HARM || a_intent == INTENT_DISARM || a_intent == INTENT_GRAB))
```

Comment:
Are you sure about including `INTENT_DISARM` and `INTENT_GRAB`?

Going to be pretty annoying to toggle bump off just to bully hosts that break out of nests when they go running for an exit.

Reply:
the problem is not even with bump, bump is slow and all, problem is that they when on not bump just literally zooms the human different than other xenos, ideally they should get more slowdown when pushing but, needs some other coding to do, i'll see if i can. also missing a parenthesis, will warn blase

Reply:
Queens literally can't push a human around no matter what now nor can any other caste not just `CASTE_CANT_PUSH ` ones. Is that the intent? Splinter says there may just be parentheses missing, but aside from the runner and hunter I still think xenos should be able to push humans.
![G2ffepXx1k](https://user-images.githubusercontent.com/64715958/95669614-85c02100-0b37-11eb-83d7-4138c81591a6.gif)

Reply:
Also if we do remove pushing the bump attack button now lies to you when you toggle it off: `You will now push enemies who are in your way.`

Reply:
yeah bug, also when bump is off should just be, "You will now not attack enemies on your way"

Reply:
since on bump you push as well(but slower)